themes/default/assets/sass/_header.scss

```diff
@@ -86,12 +83,24 @@
         @apply hidden;
 
         @screen lg {
-            @apply flex list-none justify-center;
-            width: calc((100% / 7) * 5);
+            @apply flex items-center flex-grow justify-evenly mx-8;
```

Comment:
The changes in this file are just to fine-tune the main nav on desktop and mobile.